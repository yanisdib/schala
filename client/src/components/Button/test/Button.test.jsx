import { shallow } from 'enzyme';

import { Button } from '../..';


describe('Button Component', () => {
    let wrapper;
    const { variant } = Button.defaultProps;

    beforeEach(() => wrapper = shallow(<Button variant="primary">Button</Button>))

    it('renders properly', () => expect(wrapper).toMatchSnapshot())

    it('renders with default props properly', () => expect(variant).toEqual('primary'))

    it('renders with children text properly', () => expect(wrapper.text()).toEqual('Button'))
})