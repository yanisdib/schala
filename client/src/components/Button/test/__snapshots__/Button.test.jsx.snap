// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component renders properly 1`] = `
<button
  className="sc-bdfBQB kA-djzH"
>
  Button
</button>
`;
